### ISSUE TITLE
Wrong font selected in small phones

### ISSUE THREAD

**Issue by alrajdev** [ottop/yam_launcher#10]
In Pixel 5 simulator when we select a font, the font below it, is selected. In the screen recording, you can see when selecting 'Sans Serif' 'Sans Serif Light' is selected instead.

This could be related to the popup taking the whole screen height. If you notice, the popup is resizing in the bottom after the popup is already open.

The other dropdowns work correctly since they are small in height.
The font selection works in large phone like Pixel 8 Pro where the list doesn't take up the whole screen height.

Pixel 5

https://github.com/user-attachments/assets/c9e5d28b-4ec8-41b3-9ff1-3b0b15ce99f5

Pixel 8 Pro

https://github.com/user-attachments/assets/411b0607-0568-480d-8476-9bcebb1049b6

**Comment by ottop** [comment]
I'll look into whether I can reproduce this and if so, I'll try other methods of making the menu and see if they work better.

Currently on vacation, so I'll be looking at your pull requests and this in a week or 2.

**Comment by ottop** [comment]
> The issue is also present in the app shortcut selection. My Pixel 5 phone also reproduces the same issue, not just the simulator.
> 
> But the issue doesn't reproduce in your XZ2 compact and that is a weird situation, since it is smaller than Pixel 5.

Yeah, it's strange. I chose the current system to fix a similar issue that I could reproduce. I think I might just have to make my own custom menu to make that work because Android's built-in ones seem to mostly all cause problems.

**Comment by ThomasNowProductions** [comment]
Hi there,

Since it’s been a while since you opened this issue, I’d like to check in. Is the problem still occurring in the latest version, or do you still want this feature? I want to make sure every change continues to benefit our community.

If I don’t hear back within a week, I’ll go ahead and close this issue as *not planned*. Of course, you’re always welcome to reopen or create a new issue if needed.

Thanks for your contribution!
